src/tests/ftest/pool/rebuild_with_io.py

```diff
@@ -36,7 +36,6 @@ class RebuildWithIO(TestWithServers):
     :avocado: recursive
     """
 
-    @skipForTicket("DAOS-2922")
```

Comment:
This worked locally or in Ci ? is it not going to block now because of DAOS-5611?

Reply:
Enabling the test here to see if it fails becasue of DAOS-5611. I don't want to assume all tests originally skipped for DAOS-2922 are failing for DAOS-5611. I will push another commit to skip this test if it fails.